tutor/utils.py

```diff
@@ -162,12 +163,25 @@ def docker(*command: str) -> int:
     return execute("docker", *command)
 
 
+@lru_cache
+def _docker_compose_command() -> str:
+    """
+    A helper function to determine which program to call when running docker compose
+    """
+    if os.environ.get("TUTOR_USE_COMPOSE_SUBCOMMAND") is not None:
```

Comment:
Could you add a mention of this environment variable in the docs? (in local.rst for instance) Please make it clear that Docker Compose v2 is still in test mode, and we are not quite sure yet whether we are fully compatible.